examples/todo/src/dsync.rs

```diff
@@ -0,0 +1,26 @@
+use dsync::{GenerationConfig, TableOptions};
+use std::path::PathBuf;
+
+fn main() {
+    let dir = env!("CARGO_MANIFEST_DIR");
+
+    let schema_file = PathBuf::from_iter([dir, "src/schema.rs"]);
+    let models_dir = PathBuf::from_iter([dir, "src/models"]);
+
+    dsync::generate_files(
+        schema_file.as_path(),
+        models_dir.as_path(),
+        GenerationConfig {
+            connection_type: "diesel::pg::PgConnection".into(),
+            default_table_options: TableOptions::default().disable_serde(),
```

Comment:
maybe we should also use `single_model_file` mode for this example - this should make the directory structure more clear (in this small example)

Reply:
I somewhat disagree with using `single_model_file` in the basic example because it's not the default and we're not thinking of making that the default behaviour. That being said, I've made the change anyway as it does simplify the directory structure.